manifest.json

```diff
@@ -0,0 +1,140 @@
+{
+  "repos": [
+    {
+      "name": "microsoft/dotnet-framework",
+      "readmePath": "README.dotnet-framework.md",
+      "images": [
+        {
+          "sharedTags": {
+            "4.7.1": {},
```

Comment:
Clarifying question - Tags will be in descending order and hence there is no need for `readmeOrder` , right?

Reply:
If `readmeOrder` is not specified, then the manifest order is used.  The only scenario in which I would recommend specifying the `readmeOrder` is when the build order is different than your desired order in the readme.